assets/js/index.js

```diff
@@ -23,17 +26,25 @@ async function postTodo(todoText) {
   }
 }
 
-addTaskIcon.addEventListener("click", function () {
-  postTodo(addTaskInput.value);
+addTaskIcon.addEventListener("click", async function () {
+  await postTodo(addTaskInput.value);
   clearAddTodoInput();
   clearErrorMessage();
+  const tasks = await getTasks();
+  showTasks(tasks);
+  filterItemAll.innerText = tasks.length;
+  filterItemUncompleted.innerText = tasks.length;
```

Comment:
Move these to a stand alone function and just call the function here.

Reply:
+ This code won't work. Please write a logic to filter them out. Bring back the default tasks in the back-end code. And write the filter logic.

Reply:
➕ Add Logic of completed tasks. It's not handled here.

Reply:
Done.